src/Amp/Database/Datasource.php

```diff
@@ -175,6 +175,16 @@ function updateHost() {
     }
   }
 
+  /**
+   * Convert a PDO driver name to a PHP extension name.
+   */
+  function convertPDODrivertoPHPExtension($driver) {
+    if ($driver == 'mysql') {
```

Comment:
In terms of how `CreateCommand` actually sets up the DB... it should be using... PDO. IIRC, that means the `mysql`-vs-`mysqli` shouldn't matter.

(In particular, that work is delegated to `Amp\Database\MySQL::createDatabase()` which calls `$this->adminDatasource->createPDO()`.)